index.php

```diff
@@ -1 +1,24 @@
 <?php
+require_once("functions.php");
+
+$dbPDO = connectDb('db', 'root', 'password', 'collectorapp');
```

Comment:
dbPDO is a name that kinda repeats itself - database PHP data object - not a huge problem though

Reply:
i would suggest in order to make this more like industry standard code, at the top of the file create constants for host, username, password and dbname and then pass those as params. nothing particularly wrong with this but that would be more like real world code

Reply:
Done